src/lambda/commands/createNewSamApp.ts

```diff
@@ -41,7 +41,9 @@ export async function resumeCreateNewSamApp(context: Pick<vscode.ExtensionContex
     }
 }
 
-export async function createNewSamApp(context: Pick<vscode.ExtensionContext, 'globalState'>): Promise<void> {
+export async function createNewSamApp(
+    context: Pick<vscode.ExtensionContext, 'globalState'>
+): Promise<Map<string, string> | undefined> {
     const config = await new CreateNewSamAppWizard().run()
     if (!config) {
         return
```

Comment:
Nit: Style-wise, we prefer to explicitly return `undefined` instead of using empty return statements.

Reply:
:+1: